### PR TITLE
Docs: Update Monika Documentation

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. This PR solves #381 

## How did you implement / how did you fix it  
1.  Improved Monika documentation according to the ticket

## How to test  
1. Run `cd docs && npm run dev`
2. Check if the content matches

## Additional information

Configuration examples in Installation section (which is now removed) is now in Guides > Notification

### CODE REVIEW

File: docs/src/pages/quick-start.md [hunk 1]
```diff
@@ -3,169 +3,56 @@ id: quick-start
...
-3. [SendGrid](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.sendgrid.example.json)
-4. [Webhook](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.webhook.example.json)
-5. [Slack](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.slack.example.json)
-6. [Telegram](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.telegram.example.json)
-7. [WhatsApp](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.whatsapp.example.json)
-8. [Microsoft Teams](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.teams.example.json)
-9. [Monika Whatsapp Notifier](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.monika-whatsapp.example.json)
-10. [Desktop Notifications](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.desktop.example.json)
-11. [Default](https://github.com/hyperjumptech/monika/blob/main/monika.example.json)
+You can install Monika two methods:
```
Comment (gsukanto) [inline docs/src/pages/quick-start.md:10]: `Two ways to install Monika:`

File: docs/src/pages/quick-start.md [hunk 2]
```diff
@@ -3,169 +3,56 @@ id: quick-start
...
-    1 Socket in /run/screen/S-server.
-  ```
+Congratulations, you have successfully run Monika in your machine!
 
-  `9049.pts-0.the-server` is the name of the screen.
+## Next Step
 
-- Then run `screen -r <name_of_the_screen>`.
-- To stop Monika, hit Ctrl+c then Ctrl+d.
+For advanced configuration such as configuring notifications, probes, and alerts, you can find them on the sidebar menu for more details.
```
Comment (gsukanto) [inline docs/src/pages/quick-start.md:58]: `For advanced configuration such as configuring notifications, probes, and alerts, you can find them on the sidebar menu.`

File: docs/src/pages/tutorial/run-in-background.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ```
+  There is a screen on:
+    9049.pts-0.the-server	(03/23/21 08:34:38)	(Detached)
+    1 Socket in /run/screen/S-server.
+  ```
+
+  `9049.pts-0.the-server` is the name of the screen.
+
+- Then run `screen -r <name_of_the_screen>`.
+- To stop Monika, hit Ctrl+c then Ctrl+d.
```
Comment (gsukanto) [inline docs/src/pages/tutorial/run-in-background.md:39]: `Ctrl-d` is for exit/terminate Screen.

**Review by gsukanto — APPROVED** [review]

File: docs/src/pages/tutorial/run-in-background.md (hunk 3 above)
Comment (dennypradipta) [reply docs/src/pages/tutorial/run-in-background.md:39]: So it should be "To stop Monika, hit CTRL+C and then CTRL+D to exit/terminate Screen"?

**Review by sapiderman — APPROVED** [review]
looks great, some minor feedback.

File: docs/src/pages/quick-start.md [hunk 4]
```diff
@@ -3,169 +3,56 @@ id: quick-start
...
 
-    INFO_TYPE: current-time iteration probe:id _probeURL_ [response time in ms],[ALERT messages if any], [NOTIFICATION messages if any]
-   ```
+Download the example configuration that uses Desktop notification [here](https://raw.githubusercontent.com/hyperjumptech/monika/main/config_sample/config.desktop.example.json) and rename it as monika.json
 
-## Configuration file
+## Run Monika
 
-> The configuration file contains the [notification](/monika/guides/notifications), [probes](/monika/guides/probes), and [alerts](/monika/guides/alerts) as shown below.
+Monika by default reads a configuration file called `monika.json` in the current working directory if it exists. To simply run previously downloaded the configuration, run this command on the Terminal in the same path you saved `monika.json` file:
```
[the remark this answers is not in the document]
Comment (nicnocquee) [reply docs/src/pages/quick-start.md:36]: Change "To simply run previously downloaded the configuration, run this command on the Terminal in the same path you saved `monika.json` file:" to

To run monika with the configuration file that you have downloaded before, run this command in the Terminal from the directory that contains the monika.json file:

File: docs/src/pages/quick-start.md [hunk 5]
```diff
@@ -3,169 +3,56 @@ id: quick-start
...
-    1 Socket in /run/screen/S-server.
-  ```
+Congratulations, you have successfully run Monika in your machine!
 
-  `9049.pts-0.the-server` is the name of the screen.
+## Next Step
 
-- Then run `screen -r <name_of_the_screen>`.
-- To stop Monika, hit Ctrl+c then Ctrl+d.
+For advanced configuration such as configuring notifications, probes, and alerts, you can find them on the sidebar menu.
```
Comment (nicnocquee) [inline docs/src/pages/quick-start.md:58]: better to directly put the links in this sentence, e.g. `[notifications](/path/to/notifications)`

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Comment by dennypradipta** [comment]
@nicnocquee @sapiderman should be fixed by now. Please re-review :+1:

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**